requirements/conda.txt

```diff
@@ -25,3 +25,4 @@ rtree
 sphinx
 spyder-kernels >=1.7.0
 watchdog
+intervaltree
```

Comment:
Not alphabetized.

Reply:
I agree. @andfoy, please put this new dependency in alphabetical order among the other ones.